doc/modules/metric_exporter.md

```diff
@@ -5,7 +5,7 @@ title: Metric exporter
 
 # Metric exporter
 
-Metric exporter collects statistics from Rspamd and feeds them to external monitoring/graphing systems. [Graphite](https://graphiteapp.org/) is the only supported backend for now.
```

Comment:
It's fine to call out support for Prometheus metrics here (perhaps in a new paragraph) - but it's wrong to claim that Prometheus is supported in metric exporter.

Reply:
I also agree with this point: prometheus endpoint is a `pull` endpoint, whilst `metric_exporter` utilises a `push` model for all operation. We can definitely mention prometheus support, but it should just be a link to the relevant `controller` worker documentation.